NdagiStanley/vue-django
# 2.0

#### What does this PR do?
Fixes #12 
#### Description of Task to be completed?
Update the boilerplate to include `vue2`
Now running `vue init NdagiStanley/vue-django my-project` - has `Vue 2`
and running `vue init NdagiStanley/vue-django#1.0 my-project` - has `Vue 1.x` (Soon to be deprecated)
#### How should this be manually tested?
Run 
```
$ vue init NdagiStanley/vue-django my-project
$ cd my-project
$ npm install
$ npm dev
```

In the VueJS dev-tools, you should see this:
![image](https://user-images.githubusercontent.com/15629602/33137116-14b7e51e-cfb8-11e7-8372-f4d4b38a75d1.png)

#### Any background context you want to provide?
Thanks to the commenters on this issue for raising this up #12 .
